VALUES-CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,19 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
+## Values
+- ScaLabs is both labs and people
+- Functioning societies produce wonder
+- Human rights are universal
+- Good is sustainable
+- Less but better
+- Form follows the content
+- Committment to highest standards defines us
+
+Topics:
+- Serve
+- Respect
+- Conflict resolution
+- Dealing the competitor 
+- ...
```

Comment:
How to communicate and behave with colleagues, clients, and external partners is really important and should be the one of the main topics here

Reply:
How about we extend the Respect with those items as sub-gategories like this?:
```
Respect:
our colleagues
--1.
--2.
our clients
-communication
--1.
--2.

```

Reply:
Could voice and tone guidelines be part of this?